docs/how-to-use-minio-server-side-encryption.md

```diff
@@ -0,0 +1,65 @@
+# How to use Minio's server-side-encryption [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
```

Comment:
Header can be `How to use Minio's server-side-encryption with aws-cli`.

Reply:
I would agree but I will add e.g. mc too - mc does not support  SSE-C yet...
I think we can avoid a separate doc for aws-cli, mc a.s.o.

Reply:
Cookbook has docs for specific use-cases, in that sense a separate doc on using mc for SSE will be the right approach..

Reply:
IMO, we could have a doc in server docs (https://github.com/minio/minio/tree/master/docs) with general  info about SSE. In cookbook, we can simply link to the info doc and then focus on the specific use-case.

Reply:
Okay, than I change title and will also open PR for server doc.
When that PR is merged I replace SSE info here with link?!

Reply:
perfect, thanks!